entries/a/aidvantage.studentaid.gov.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Aidvantage": {
+    "domain": "aidvantage.studentaid.gov",
+    "img": "aidvantage.studentaid.gov.png"
```

Comment:
```suggestion
    "img": "aidvantage.studentaid.gov.png",
```

Reply:
Already fixed in my force-pushed commit 😬